js/app/Visualization/UI/style.less

```diff
@@ -109,26 +109,24 @@ body {
 .loading {
   padding-left: 14px;
   padding-top: 10px;
+  position: absolute;
+  top: 175px !important;
 }
 
+.zoomButtons {
```

Comment:
@enriquetuya here is where we force the absolute positioning on to the google maps zoom buttons.  Also note the positioning of the  "loading spinner" in the .loading class above